### PR TITLE
fixed some typos

### PR DESCRIPTION
**Pull request by cordoval**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,13 +51,13 @@ Installation
...
 
-            ./bin/vendors install
+            php bin/vendors install
 
       * Using git submodules.
 
             $ git submodule add git://github.com/FriendsOfSymfony/FOSFacebookBundle.git vendor/bundles/FOS/FacebookBundle
             $ git submodule add git://github.com/facebook/php-sdk.git vendor/facebook
-
+
```
Comment (jmikola) [inline README.md]: It looks like you're adding trailing whitespace here.

Comment (cordoval) [inline README.md]: yeah rats, sorry, i have been so frustrated lately, nothing works

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ For step 1. and 2. there are two options:
 
   1. Select "OAuth Migration" in the Facebook application settings.
   2. Add a Facebook login button, this approach requires JS code to handle step 3.
-  3. Letting the FOSFacebookBundle redirecting to the Facebook login page
+  3. Letting the FOSFacebookBundle redirect to the Facebook login page
```
Comment (jmikola) [inline README.md]: "FOSFacebookBundle" is a proper noun here, so I don't even think the leading "the" is necessary. Also, we should add a period after this sentence to be consistent with the first and second bullets.

File: README.md [hunk 3]
```diff
@@ -51,13 +51,13 @@ Installation
 
         Run the vendors script:
 
-            ./bin/vendors install
+            php bin/vendors install
```
Comment (jmikola) [inline README.md:54]: This bundle doesn't even have a `bin/` directory. Perhaps this should be `vendor/vendors install`? I assume it's referring to [this file](https://github.com/FriendsOfSymfony/FOSFacebookBundle/blob/master/vendor/vendors.php).

Also, the shell script leads with `#!/usr/bin/env php`, so `php` is not necessary to invoke it. I would suggest we change the file permissions for the script to 755, though. It's currently 644.

Comment (cordoval) [inline README.md:54]: you are missing the point here jmikola, bin/vendors is the apps, to update the deps to download the bundle repo

File: README.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ https://developers.facebook.com/docs/guides/web/
 
 Please also refer to the official documentation of the SecurityBundle, especially
 for details on the configuration:
-http://symfony.com/doc/2.0/book/security/authentication.html
+http://symfony.com/doc/current/book/security/authentication.html
```
Comment (cordoval) [inline README.md]: this link here is not right, i think it points to a cookbook entry maybe

File: README.md (hunk 3 above)
Comment (jmikola) [inline README.md:54]: Got it. My point is the same with regards to [bin/vendors](https://github.com/symfony/symfony-standard/blob/master/bin/vendors) in the symfony-sandbox package, though. And that file has the correct permissions of 755.

Comment (stof) [inline README.md:54]: except that the syntax `php bin/vendors` works for all people whereas `./bin/vendors` does not for Windows users. This is the reason why the Sf2 documentation always uses `php`

**Comment by lsmith77** [comment]
ping ..

**Comment by jmikola** [comment]
Was that ping asking for a rebase? :)

**Comment by cordoval** [comment]
oh sorry i did not understand it, thanks for the clarification @jmikola 
please can you confirm so i can rebase @lsmith77 thanks

I feel like I am learning the github lingo

**Comment by stof** [comment]
@cordoval both rebasing and fixing stuff asked in the comments

**Pull request merged**